packages/headless-styles/src/utils/helpers.ts

```diff
@@ -39,6 +39,13 @@ export function createSvelteObj(classname = '') {
   return { class: classname }
 }
 
+export function createCSSObj(className: string, addtionalProps?: StyleProps) {
+  return {
+    className,
+    ...addtionalProps,
+  }
+}
```

Comment:
Typo on `addtionalProps`

```suggestion
export function createCSSObj(className: string, additionalProps?: StyleProps) {
  return {
    className,
    ...additionalProps,
  }
}
```